### PR TITLE
Addition of user guide

### LINKED ISSUE #14: Update User Guide with v1.0 features
Need to convert the word document into Markdown format and make changes where necessary.

### PR DESCRIPTION
**Pull request by Johnson-Yee**
Fixes #14

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,42 +1,151 @@
...
+
+### 1. Check for direct bus: ```/route```
+Checks for direct bus between two locations. <br>
+Format: <code>/route <strong>location_1 /to location_2 </strong> </code> <br>
+Example of usage: <code>/route <strong> PGP /to Raffles Hall </strong></code><br>
+Expected outcome: 
+```
+The buses you can take with their intermediate stops are: 
+1. AA2
+	Kent Ridge MRT Station-> Opp University Health Centre-> University Town
```
Comment (wamikamalik) [inline docs/UserGuide.md]: no tab here
```suggestion
Kent Ridge MRT Station-> Opp University Health Centre-> University Town
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,42 +1,151 @@
...
+/routemap
+What bus routes would you like to see?
+Currently, we have two bus routes available for your viewing 
+1.AA1 
+2.AA2 
+Type the name to view:
+aa1
+Here is the AA1 route that you have requested:
+AA1
+	PGP-> Kent Ridge MRT Station-> Opp University Health Centre-> Yusof Ishak House-> Central Library-> Kent Ridge-> Museum-> University Town-> University Health Centre-> Opp Kent Ridge MRT station-> PGPR
```
Comment (wamikamalik) [inline docs/UserGuide.md]:
```suggestion
PGP-> Kent Ridge MRT Station-> Opp University Health Centre-> Yusof Ishak House-> Central Library-> Kent Ridge-> Museum-> University Town-> University Health Centre-> Opp Kent Ridge MRT station-> PGPR
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1,42 +1,151 @@
...
+```
+
+### 4. List all bus available in NUS ```/allbus```
+Lists all bus available in NUS. <br>
+Example of usage: <code>/allbus</code> <br>
+Expected outcome:
+```
+The buses available in NUS are: 
+AA1
+	PGP-> Kent Ridge MRT Station-> Opp University Health Centre-> Yusof Ishak House-> Central Library-> Kent Ridge-> Museum-> University Town-> University Health Centre-> Opp Kent Ridge MRT station-> PGPR
```
Comment (wamikamalik) [inline docs/UserGuide.md]:
```suggestion
PGP-> Kent Ridge MRT Station-> Opp University Health Centre-> Yusof Ishak House-> Central Library-> Kent Ridge-> Museum-> University Town-> University Health Centre-> Opp Kent Ridge MRT station-> PGPR
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -1,42 +1,151 @@
...
+AA2
+	PGP-> Kent Ridge MRT Station-> Opp University Health Centre-> University Town-> Raffles Hall-> Kent Vale-> EA-> NUS IT-> University Health Centre-> Opp Kent Ridge MRT station-> PGPR
```
Comment (wamikamalik) [inline docs/UserGuide.md]:
```suggestion
PGP-> Kent Ridge MRT Station-> Opp University Health Centre-> University Town-> Raffles Hall-> Kent Vale-> EA-> NUS IT-> University Health Centre-> Opp Kent Ridge MRT station-> PGPR
```

**Review by wamikamalik — APPROVED** [review]
Looks good to merge!

File: docs/UserGuide.md [hunk 5]
```diff
@@ -1,42 +1,150 @@
...
-
-* The `DEADLINE` can be in a natural language format.
-* The `TODO_NAME` cannot contain punctuation.  
-
-Example of usage: 
-
-`todo n/Write the rest of the User Guide d/next week`
-
-`todo n/Refactor the User Guide to remove passive voice d/13/04/2020`
+>Notes about command format:
```
Comment (EthanWong2212) [inline docs/UserGuide.md:30]:
```suggestion
There are 7 commands you can use to get bus information from Nav@NUS

>Notes about command format:  
```

Comment (EthanWong2212) [reply docs/UserGuide.md:30]: Would be good to start each section with a brief intro

File: docs/UserGuide.md [hunk 6]
```diff
@@ -1,42 +1,150 @@
...
+* [FAQ](#faq)
+* [Command Summary](#command-summary)
 
 ## Quick Start
-
-{Give steps to get started quickly}
-
-1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+1. Ensure that you have Java `11` or above installed in your computer.
```
Comment (EthanWong2212) [inline docs/UserGuide.md:21]:
```suggestion
This following steps is a guide to start running Nav@NuUS.

1. Ensure that you have Java `11` or above installed in your computer.
```

Comment (EthanWong2212) [reply docs/UserGuide.md:21]: Would be good to start each section with a brief intro

**Review by EthanWong2212 — CHANGES_REQUESTED** [review]
Would be good to have brief introductions to the sections

**Pull request merged**